<HTML>

<HEAD>
<TITLE>HiCOMB 2009</TITLE>
<!doctype html public "-//w3c//dtd html 4.0 transitional//en">
<META content="text/html; charset=iso-8859-1" http-equiv=Content-Type>
<LINK REL="SHORTCUT ICON" HREF="http://www.hicomb.org/favicon.ico">
</HEAD>

<BODY background="helix.gif">

<center>
<b><font size=+2>&nbsp;
<font color="#000080">HiCOMB 2009<br>
Eighth IEEE International Workshop on High Performance Computational Biology</font>
</font></b> 

<p>
<font color="#000080">
<b><font size=+2>Monday, May 25, 2009</font></b> <br>
</font>

<font size=+2>
<font color="#000080">
<b>Rome, Italy</b> <br>
<img src="ipdps2009.jpg"><br>
<b>(Held in conjunction with the <a href="http://www.ipdps.org/">International Parallel and 
Distributed Processing Symposium</a>)</b> 
</font> 
</font> 
</center>

<hr>

<p>
<font color="#000080">
<h2><a href="proceedings.html" target="_blank">HiCOMB Online Proceedings</a></h2>
<p> 
<h2> <a href="./HiCOMB2009-SCHEDULE.txt" target="_blank">Advance Program for HiCOMB 2009</a></H2>
<p>
<h2> <a href="http://www.ipdps.org/ipdps2009/2009_registration_details.html" target="_blank">Register Online</a></h2>
(Please note that the IPDPS 2009 registration includes access to this workshop, and that there is no separate workshop registration).

<hr>

<font color="#000080">
<h2>HiCOMB 2009 Keynote Talk</h2>
<p>

<h3>De Novo Modeling of GPCR Class A Structures</h3>

<a href="http://brooks.chem.lsa.umich.edu">Charles L. Brooks III</a><br>
  University of Michigan, Ann Arbor <br>

<p>
<b>Abstract</b>: <br>

In this talk I will describe recent work to develop novel methods to
model GPCR structures from their sequence information and
statistically significant side chain contacts within a "template"
structure. Our approach utilizes methods of bioinformatics to identify
likely high confidence side chain side chain TM helical contacts and
then reconstitutes the seven TM helical domain through a simulated
annealing protocol with refinement using replica exchange and an
implicit solvent/implicit membrane sampling scheme. Results will be
presented for de novo prediction of the b2 adenergic receptor, the
adenine receptor and a number of other amine receptors.

<p>
<b>Speaker Biography</b>: <br>

Charles L. Brooks III is the Warner-Lambert Professor of Chemistry and
Professor of Biophysics at the University of Michigan. Prior to moving
to Michigan in 2008, Professor Brooks was professor of Molecular
Biology at The Scripps Research Institute since 1994 and professor of
Chemistry at Carnegie Mellon University until 1993, where he began as
an assistant professor in 1985. Professor Brooks received his
undergraduate training in chemistry, physics and mathematics from Alma
College. He studied chemical physics at Purdue University, where he
received his Ph.D. in 1982, and was a postdoctoral fellow at Harvard
University from 1982-85. Professor Brooks' research focuses on the
applications of statistical mechanics, quantum chemistry, and
computational methods to solve chemically and physically oriented
problems in biology.

</font>

<hr>
<font color="#000080">
<H2>Papers to be presented at HiCOMB 2009:</H2>
<P>

<ul>
   <li><i>Parallel Reconstruction of Neighbor-Joining Trees for Large Multiple Sequence Alignments using CUDA </i>
   <br>Yongchao Liu, Bertil Schmidt, and Douglas Maskell 
   </li>
   <li><i>Accelerating Error Correction in High-Throughput Short-Read DNA Sequencing Data with CUDA </i>
   <br>Haixiang Shi, Bertil Schmidt, Weiguo Liu, and Wolfgang Mueller-Wittig 
   </li>
   <li><i>Parallel Monte Carlo Study on Caffeine-DNA Interaction in Aqueous Solution </i>
   <br>Michael Kalugin and Alexandr Teplukhin 
   </li>
   <li><i>Dynamic Parallelization for RNA Structure Comparison </i>
   <br>Eric Aubanel, Patricia Evans, and Eric Snow 
   </li>
   <li><i>HMMer Acceleration Using Systolic Array Based Reconfigurable Architecture </i>
   <br>Yanteng Sun, Peng Li, Guochang Gu, Yuan Wen, Yuan Liu, and Dong Liu 
   </li>
   <li><i>A Resource-Efficient Computing Paradigm for Computational Protein Modeling Applications </i>
   <br>Yaohang Li, Dougalas Wardell, and Vincent Freeh 
   </li> 
   <li><i>Exploring FPGAs for Accelerating the Phylogenetic Likelihood Function </i>
   <br>Nikolaos Alachiotis, Euripides Sotiriades, Apostolos Dollas, and Alexandros Stamatakis 
   </li>
   <li><i>Long Time-scale Simulations of in vivo Diffusion using GPU Hardware </i>
   <br>Elijah Roberts, John Stone, Leonardo Sepulveda, Wen-Mei Hwu, and Zaida Luthey-Schulten 
   </li>
   <li><i>An Efficient Implementation Smith Waterman Algorithm on GPU using CUDA, for Massively Parallel Scanning of Sequence Databases </i>
   <br>Lukasz Ligowski and Witold Rudnicki 
   </li> 
   <li><i>A Novel Framework for Modeling Ion Channels </i>
   <br>May Siksik and Vikram Krishnamurthy 
   </li>
   <li><i>High-throughput Protein Structure Determination using Grid Computing </i>
   <br>Jason W. Schmidberger, Blair Bethwaite, Colin Enticott, Mark A. Bate, Steve G. Androulakis, Noel Faux, Cyril F. Reboul, Jennifer M. N. Phan, James C. Whisstock, Wojtek J. Goscinski, Slavisa Garic, David Abramson, and Ashley M. Buckle 
   </li>
   <li><i>Folding@home: Lessons From Eight Years of Volunteer Distributed Computing </i>
   <br> Adam L. Beberg, Guha Jayachandran, Siraj Khaliq, Vijay S. Pande  </li
</ul>

<hr>

<h2><font color="#000080">HiCOMB 2009 Call For Papers</font></h2>

<p>
<font color="#000080">
Computational Biology is fast emerging as an important discipline for
academic research and industrial application. The large size of
biological data sets, inherent complexity of biological problems and
the ability to deal with error-prone data all result in large run-time
and memory requirements. The goal of this workshop is to provide a
forum for discussion of latest research in developing high-performance
computing solutions to problems arising from molecular biology. We are
especially interested in parallel algorithms, memory-efficient
algorithms, large scale data mining techniques, and design of
high-performance software. The workshop will feature contributed
papers as well as invited talks from reputed researchers in the field.
</font> 

<p>
<font color="#000080">Topics of interest include but are not limited to: <br>
</font> 

<ul>
  <li><font color="#000080">Bioinformatic databases</font> 
  <li><font color="#000080">Computational genomics</font> 
  <li><font color="#000080">Computational proteomics</font> 
  <li><font color="#000080">DNA assembly, clustering, and mapping</font> 
  <li><font color="#000080">Gene expression and microarrays</font> 
  <li><font color="#000080">Gene identification and annotation</font> 
  <li><font color="#000080">Parallel algorithms for biological analysis</font> 
  <li><font color="#000080">Parallel architectures for biological applications</font> 
  <li><font color="#000080">Molecular evolution</font> 
  <li><font color="#000080">Molecular sequence analysis</font> 
  <li><font color="#000080">Phylogeny reconstruction algorithms</font> 
  <li><font color="#000080">Protein structure prediction and modeling</font> 
  <li><font color="#000080">String data structures and algorithms</font> 
</ul>

<h2>Submission guidelines:</h2> 

<p>
<font color="#000080">
Papers reporting on original research (both theoretical and
experimental) in all areas of bioinformatics and computational biology
are sought. Surveys of important recent results and directions are
also welcome. <b>To submit a paper, upload a postscript or PDF copy of
the
paper <a href="http://edas.info/newPaper.php?c=7001&submit=0&">here</a>.</b>
The paper should not exceed 12 single-spaced pages (US Letter or A4
size) in 11pt font or larger. All papers will be reviewed. IEEE CS
Press will publish the IPDPS symposium and workshop abstracts as a
printed volume. The complete symposium and workshop proceedings will
also be published by IEEE CS Press on CD-ROM and will also be
available in the IEEE Digital Library.

<!--
<p>
Authors of selected papers from the workshop will be invited to
submit extended versions of their papers for publication in a
special issue of 
a reputed journal. 
<a href="http://www.computer.org/tpds/">
<i>IEEE Transactions on Parallel and Distributed Systems</i>
</a>
on High-Performance Computational Biology.
-->

</font>

<h2>Important Dates</h2>

<table cols=2 border=1 width=500>
  <tbody>
  <tr>
    <TD><font color="#000080">Workshop Paper Due: </font></TD>
    <TD><font color="#000080"><b>November 30, 2008 (Extended Deadline)</b></font></TD>
  </tr>
  <tr>
    <TD><font color="#000080">Author Notification:</font></TD>
    <TD><font color="#000080">December 25, 2008</font></TD>
  </tr>
  <tr>
    <TD><font color="#000080">Camera-ready Paper Due:</font></TD>
    <TD><font color="#000080">February 15, 2009</font></TD>
  </tr>
  </tbody>
</table>

<hr>

<h2>Workshop Co-Chairs</h2>

<table cols=2 width=677>
  <tbody>
  <tr>
    <td valign=top><font color="#000080">
      <a href="http://vulcan.ee.iastate.edu/~aluru/home.html">Srinivas Aluru</a> <br>
      Dept. of Electrical &amp; Computer Engg. and <br>
      Lawrence H. Baker Center for Bioinformatics <br>
      &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; &amp; Biological Statistics <br>
      Iowa State University <br>
      3227 Coover Hall <br>
      Ames, IA 50011, USA <br>
      Phone: +1.515.294.3539<br>
      Email:
      <script type="text/javascript">
      <!-- 
	          varE=('iastate.edu')
		         varE=('aluru' + '@' + varE)
			        document.write('<A href="mailto:' + varE + '">' + varE + '</a>')
      //-->
      </script> 
      </font>
    </td>
    <td valign=top><font color="#000080">
      <a href="http://www.cc.gatech.edu/~bader">David A. Bader</a><br>
      College of Computing <br>
      Georgia Institute of Technology <br>
      Atlanta, GA 30332  USA <br>
      Phone: +1.404.894.3152 <br>
      Email: 
      <script type="text/javascript">
      <!-- 
	          varE=('cc.gatech.edu')
		         varE=('bader' + '@' + varE)
			        document.write('<A href="mailto:' + varE + '">' + varE + '</a>')
      //-->
      </script> 
      </font>
     </td>
  </tr>
  </tbody>
</table>

<h2>Program Chair</h2>

<table cols=1 width=677>
  <tbody>
  <tr>
  <td valign=top><font color="#000080">
  <b><a href="http://www.cis.udel.edu/~taufer">Michela Taufer</a></b><br>
  Dept. of Computer and Information Sciences <br>
  University of Delaware <br>
  103 Smith Hall <br>
  Newark, DE 19716 USA  <br>
  Phone: +1.302.831.0071 <br>
  Email:
      <script type="text/javascript">
      <!-- 
	          varE=('acm.org')
		         varE=('taufer' + '@' + varE)
			        document.write('<A href="mailto:' + varE + '">' + varE + '</a>')
      //-->
      </script> 
  </font>
  </td>
  </tr>
  </tbody>
</table>

<h2>Program Committee</h2>
<ul>
<li>Dmitry A. Afonnikov, Institute of Cytology and Genetics SB RAS, 
<br>Novosibirsk and Novosibirsk State University (Russia)
<li>Pratul K. Agarwal, Oak Ridge National Laboratory (USA)
<li>Nancy Amato, Texas A&M University (USA)
<li>Roger Armen, University of Michigan at Ann Arbor (USA)
<li>Georg Fuellen, Ernst-Moritz-Arndt-University Greifswald (Germany)
<li>Concettina Guerra, Georgia Tech (USA) and University of Padova (Italy)
<li>Ming-Ying Leung, University of Texas at El Paso (USA)
<li>Mark Miller, San Diego Supercomputer Center (USA)
<li>Aiichiro Nakano, University of South California (USA)
<li>Sandeep Patel, University of Delaware (USA)
<li>Geppino Pucci, University of Padova (Italy)
<li>Jacques Rougemont, EPFL (Switzerland)
<li>Alexandros Stamatakis, Ludwig-Maximilians-University Munich (Germany)
<li>Bertil Schmidt, Nanyang Technological University (Singapore)
<li>Heiko Schroder, RMIT University (Australia)
<li>John Stone, University of Illinois at Urbana-Champaign (USA)
<li>Wing-Kin Sung, National University of Singapore (Singapore)
<li>Jim Tisdall, DuPont (USA)
<li>Chau-Wen Tseng, University of Maryland, College Park (USA)
</ul>

<p>
<font color="#000080">For up-to-date information about this workshop, please visit
<a href="http://www.hicomb.org/">http://www.hicomb.org/</a>. <br>
</font> 
</p>

<hr>

<h2>
Call for Papers for HiCOMB 2009: </h2>
<ul>
<li> <a href="http://www.hicomb.org/HiCOMB2009/HiCOMB2009-CFP.txt">ASCII Format</a>
<li> <a href="http://www.hicomb.org/HiCOMB2009/HiCOMB2009-CFP.pdf">PDF Format</a>
</ul>

<hr>

<h2>
Previous meetings
</h2>
<ul>
<li> <a href="http://www.hicomb.org/HiCOMB2008/">
     7th International Workshop on High Performance Computational Biology (HiCOMB 2008)
     </a>
<li> <a href="http://www.hicomb.org/HiCOMB2007/">
     6th International Workshop on High Performance Computational Biology (HiCOMB 2007)
     </a>
<li> <a href="http://www.hicomb.org/HiCOMB2006/">
     5th International Workshop on High Performance Computational Biology (HiCOMB 2006)
     </a>
<li> <a href="http://www.hicomb.org/HiCOMB2005/">
     4th International Workshop on High Performance Computational Biology (HiCOMB 2005)
     </a>
<li> <a href="http://www.hicomb.org/HiCOMB2004/">
     3rd International Workshop on High Performance Computational Biology (HiCOMB 2004)
     </a>
<li> <a href="http://www.hicomb.org/HiCOMB2003/">
     2nd International Workshop on High Performance Computational Biology (HiCOMB 2003)
     </a>
<li> <a href="http://www.hicomb.org/HiCOMB2002/">
     1st International Workshop on High Performance Computational Biology (HiCOMB 2002)
     </a>
</ul>

<hr>

<font size=3>
<i>
<script language="JavaScript">
<!---//hide script from old browsers
document.write( "Last updated: "+ document.lastModified );
          //end hiding contents --->
</script>
</i>

</BODY>
</HTML>
