<HTML>

<HEAD>
<TITLE>International Workshop on High Performance Computational Biology (HiCOMB)</TITLE>
<!doctype html public "-//w3c//dtd html 4.0 transitional//en">
<META content="text/html; charset=iso-8859-1" http-equiv=Content-Type>
<LINK REL="SHORTCUT ICON" HREF="http://www.hicomb.org/favicon.ico">
</HEAD>

<BODY background="helix.gif">

<CENTER><B><FONT size=+2>&nbsp;<font color="#000080">HiCOMB 2004<BR>
Third IEEE International Workshop on High Performance Computational Biology
</font></FONT></B> 

<P><font color="#000080"><B><FONT size=+2>Monday, April 26, 2004</FONT></B> <BR></font>

<FONT size=+2><font color="#000080">
<B>Eldorado Hotel, Santa Fe, NM</B> <BR>
<img src="ipdps04.gif"><br>
<B>(Held in conjunction with the <A href="http://www.ipdps.org/">International Parallel and 
Distributed Processing Symposium</A>)</B> 

</font> 

<p>
</CENTER>

</FONT> 

<P>&nbsp; 



<hr>
<font color="#000080">
<H2><a href="proceedings.html" target="_blank">HiCOMB Online Proceedings</a></H2>
<P> 
     <H2> <a href="HICOMB2004-SCHEDULE.txt" target="_blank">Advance Program for HiCOMB 2004</a></H2>
<!-- <P>
     <H2> <a href="https://newton.computer.org/conferences/ip03.nsf" target="_blank">Online Registration for IPDPS</a></H2>
-->     
</font>


<hr>
<font color="#000080">
<H2>HiCOMB 2004 Keynote Talk</H2>
<P>
<H3>Three-Dimensional Reconstruction of Viruses: Challenges and Strategies</H3>

  <a href="http://bilbo.bio.purdue.edu/~baker/">Timothy S. Baker</a><br>
  Purdue University<p>

  Viruses are cellular parasites.  A deeper understanding of many
aspects of viral life cycles has emerged from coordinated molecular
and structural studies carried out with a wide range of viral
pathogens.  Structural studies of viruses by means of transmission
electron cryo-microscopy (cryo-EM), 3D image reconstruction, and
pseudo-atomic modeling methods have grown explosively in the last
decade.  These methods have been successfully employed in the
investigation of a wide range of icosahedral viruses, ranging in size
from as small as 30nm to larger than 200nm.  The talk will review the
current technologies involved in obtaining 3D structural information
on viruses and briefly highlight some recent results at better than 10
&Aring; resolution on both enveloped and non-enveloped icosahedral
viruses.  Numerous technological obstacles stand in the path of
obtaining near atomic resolution (0.4 nm or better) structural data on
symmetric as well as non-symmetric viruses.  Particular emphasis will
be devoted to describing the development of new computational tools
designed to provide high throughput data analysis at high resolution.
<p>

</font>
<hr>

<font color="#000080">
<H2>Papers to be presented at HiCOMB 2004:</H2>
<P>
<UL>

<LI><i>Study of a Highly Accurate and Fast-Protein Ligand
Docking Algorithm Based Upon Molecular Dynamics</i><br>
M. Taufer, M. Crowley, D. Price, A.A. Chien, and C. L. Brooks III 

<LI><i>Parallel Protein Folding with STAPL</i><br>
S. Thomas and N. Amato

<LI><i>A Hypergraph Model for the Yeast Protein Complex Network</i><br>
E. Ramadan, A. Tarafdar, and A. Pothen

<LI><i>Dynamic Programming for LR-PCR Segmentation of Bacterium Genomes</i><br>
R. Andonov, D. Lavenier, N. Yanev, and P. Veber

<LI><i>Parallel RNA Sequence-Structure Alignment</i><br>
T. Liu and B. Schmidt
 
<LI><i>Constrained De Novo Peptide Identification via Multiobjective Optimization</i><br>
J.M. Mallard, A. Heredia-Langner, D.J. Baxter, K.H. Jarman, and W. R. Cannon
 
<LI><i>Identification of Mycobacteriun Species Using Curated Custom Databases</i><br>
D. Kuyper, H. Ali, A.M. Mohamed, and S.H. Hinrichs

<LI><i>High Performance Computational Tools for Motif Discovery</i><br>
N.E. Baldwin, R.L. Collins, M. A. Langston, M.R. Leuze, C.T. Symons, and B. H. Voy

<LI><i>High Performance Biosequence Database Scanning on Reconfigurable Platforms</i><br>
T. Oliver and B. Schmidt
 
<LI><i>New Fast and Acurate Heuristics for Inference of Large Phylogenic Trees</i><br>
A.P. Stamatakis, T. Ludwig, and H. Meier

<LI><i>Large-Scale Simulation of Eukaryotic Cell Signaling Processes</i><br>
J.H. Miller and F. Zheng

<LI><i>Particle Swarm Optimization and Neural Network Application for QSAR</i><br>
Z. Wang, G.L. Durst, R.C. Eberhart, D.B. Boyd, and Z. Ben Miled
</UL>

</font>




<hr>
<h2><font color="#000080">HiCOMB 2004 Call For Papers</font></h2>

<P><font color="#000080">

  Computational Biology is fast emerging as an important discipline
for academic research and industrial application. The large size of
biological data sets, inherent complexity of biological problems and
the ability to deal with error-prone data all result in large run-time
and memory requirements. The goal of this workshop is to provide a
forum for discussion of latest research in developing high-performance
computing solutions to problems arising from molecular biology. We are
especially interested in parallel algorithms, memory- efficient
algorithms, large scale data mining techniques, and design of
high-performance software.  The workshop will feature contributed
papers as well as invited talks from reputed researchers in the
field.

</font> 

<P><font color="#000080">Topics of interest include but are not limited to: <BR>&nbsp;</font> 

<UL>

  <LI><font color="#000080">Bioinformatic databases</font> 

  <LI><font color="#000080">Computational genomics</font> 

  <LI><font color="#000080">Computational proteomics</font> 

  <LI><font color="#000080">DNA assembly, clustering, and mapping</font> 

  <LI><font color="#000080">Gene expression and microarrays</font> 

  <LI><font color="#000080">Gene identification and annotation</font> 

  <LI><font color="#000080">Parallel algorithms for biological analysis</font> 

  <LI><font color="#000080">Parallel architectures for biological applications</font> 

  <LI><font color="#000080">Molecular evolution</font> 

  <LI><font color="#000080">Molecular sequence analysis</font> 

  <LI><font color="#000080">Phylogeny reconstruction algorithms</font> 

  <LI><font color="#000080">Protein structure</font> 

  <LI><font color="#000080">String data structures and algorithms</font> </LI></UL>

<FONT 

size=+2>

<P><font color="#000080"><BR><B>Submission guidelines:</B> 

</font></FONT> 

<P><font color="#000080">

  Papers reporting on original research (both theoretical and
experimental) in all areas of bioinformatics and computational biology
are sought. Surveys of important recent results and directions are
also welcome. To submit a paper, send a postscript or PDF copy of the
paper by email to the workshop's Program Chair, <a href="http://www.cs.ucf.edu/~dcm">Prof. Dan C. Marinescu</a>,
<<a href="mailto:dcm@cs.ucf.edu">dcm@cs.ucf.edu</a>>. The paper should not exceed 12 single-spaced pages
(US Letter or A4 size) in 11pt font or larger. All papers will be
reviewed. IEEE CS Press will publish the IPDPS symposium and workshop
abstracts as a printed volume. The complete symposium and workshop
proceedings will also be published by IEEE CS Press on CD-ROM and will
also be available in the IEEE Digital Library.

<p>
Authors of selected papers from the workshop will be invited to
submit extended versions of their papers for publication in the
journal <a href="http://www.cc-pe.net/">Concurrency and Computation: Practice and Experience</a>.

</font><FONT size=+2> 

<P><font color="#000080"><B>Important Dates</B> <BR>&nbsp;</font> 

<TABLE cols=2 border=1 width=500>

  <TBODY>

  <TR>
    <TD><font color="#000080">Workshop Paper Due:</font></TD>
    <TD><font color="#000080">December 1, 2003</font></TD>
  </TR>

  <TR>
    <TD><font color="#000080">Author Notification:</font></TD>
    <TD><font color="#000080">December 31, 2003</font></TD>
  </TR>
  
  <TR>
    <TD><font color="#000080">Camera-ready Paper Due:</font></TD>
    <TD><font color="#000080">January 23, 2004</font></TD>
  </TR>

 </TBODY></TABLE>

<hr>
<P><font color="#000080"><B>Workshop Co-Chairs</B> <BR>&nbsp;</font> 

<TABLE cols=2 width=677>

  <TBODY>

  <TR>

    <TD valign=top><font color="#000080">
      <a href="http://www.ece.unm.edu/~dbader/">David A. Bader</a><BR>
      Electrical &amp; Computer Engr. Dept. <BR>
      University of New Mexico <BR>
      Albuquerque, NM 87131  USA <BR>
      Phone: +1.505.277.6724 <BR>
      Fax: +1.505.277.1439 <BR>
      Email: <A href="mailto:dbader@ece.unm.edu">dbader@ece.unm.edu</A></font></TD>

    <TD valign=top><font color="#000080">
      <a href="http://vulcan.ee.iastate.edu/~aluru/home.html">Srinivas Aluru</a> <BR>
      Dept. of Electrical &amp; Computer Engg. and <BR>
      Lawrence H. Baker Center for Bioinformatics <BR>
      &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; &amp; Biological Statistics <BR>
      Iowa State University <BR>
      3218 Coover Hall <BR>
      Ames, IA 50011, USA <BR>
      Phone: +1.515.294.3539<BR>
      Fax: +1.515.294.8432<BR>
      email: <A href="mailto:aluru@iastate.edu">aluru@iastate.edu</A></font></TD>

</TR></TBODY></TABLE>

<P><font color="#000080"><B>Program Chair</B> <BR>&nbsp;</font> 

<TABLE cols=1 width=677>
  <TBODY>
  <TR>
    <TD valign=top><font color="#000080">
     <a href="http://www.cs.ucf.edu/~dcm">Dan C. Marinescu</a><BR>
     Computer Science Department<BR>
     University of Central Florida<BR>
     4000 Central Florida Boulevard<BR>
     Orlando, Florida, 32816, USA<BR>
     Phone:     +1 407 823 4860<BR>
     FAX:       +1 407 823 5419<BR>
     Email: <a href="mailto:dcm@cs.ucf.edu">dcm@cs.ucf.edu</a><BR>
     <a href="http://www.cs.ucf.edu/~dcm">http://www.cs.ucf.edu/~dcm</a>
     </font></TD>
</TR></TBODY></TABLE>

<P><font color="#000080"><B>Program Committee</B> <BR>&nbsp;</font> 

<TABLE cols=3>

  <TBODY>

  <TR>

    <TD><font color="#000080">
     <a href="http://www.cs.purdue.edu/people/axa">Prof. Alberto Apostolico</a><BR>
     Computer Sciences Department<BR>
     Purdue University<BR>
     West Lafayette, IN, 47907<BR>
     Email: <a href="mailto:axa@cs.purdue.edu">axa@cs.purdue.edu</a><BR>
     <a href="http://www.cs.purdue.edu/people/axa">http://www.cs.purdue.edu/people/axa</a><BR>
     Phone: +1.765.494.6015
      </font> </TD>

    <TD valign=top><font color="#000080">
     <a href="http://www.dei.unipd.it/~guerra">Prof. Concettina Guerra</a><BR>
     Dip. Ingegneria dell'Informazione<BR>
     Universita'di Padova<BR>
     via Gradenigo 6a, 35100, Padova, Italy<BR>
     Email: <a href="mailto:guerra@dei.unipd.it">guerra@dei.unipd.it</a><BR>
     <a href="http://www.dei.unipd.it/~guerra">http://www.dei.unipd.it/~guerra</a><BR>
     Phone:  +39 049 827 7933
      </font> </TD>
 
    <TD valign=top><font color="#000080">
    <a href="http://www.cs.purdue.edu/people/spa">Prof. Wojciech Szpankowski</a><BR>
    Computer Sciences Department<BR>
    Purdue University<BR>
    West Lafayette, IN, 47907<BR>
    Email: <a href="mailto:spa@cs.purdue.edu">spa@cs.purdue.edu<a><BR>
    <a href="http://www.cs.purdue.edu/people/spa">http://www.cs.purdue.edu/people/spa</a><BR>
    Phone: +1.765.494.6703
      </font> </TD>

      </TR></TBODY></TABLE>

<P><font color="#000080">For up-to-date information about this workshop, please visit
<A href="http://www.hicomb.org/">http://www.hicomb.org/</A>. 

<BR></font> </P></FONT>

<hr>
<H2>
Call for Papers for HiCOMB 2004:
</H2>
<UL>
<LI> <a href="HiCOMB2004-CFP.txt">ASCII Format</a>
<LI> <a href="HiCOMB2004-CFP.ps">PostScript Format</a>
<LI> <a href="HiCOMB2004-CFP.pdf">PDF Format</a>
</UL>

<hr>
<H2>
Previous meetings
</H2>
<UL>
<LI> <a href="http://www.hicomb.org/HiCOMB2003/">
     2nd International Workshop on High Performance Computational Biology (HiCOMB 2003)
     </a>
<LI> <a href="http://www.hicomb.org/HiCOMB2002/">
     1st International Workshop on High Performance Computational Biology (HiCOMB 2002)
     </a>
</UL>

<hr>
Last updated:
<i>
Sat Jan 10 08:06:01 MST 2004
</i>
</BODY>
</HTML>

